AndroidBound/src/main/java/solutions/alterego/androidbound/android/BoundFragmentDelegate.java

```diff
@@ -256,7 +256,7 @@ public void onDestroyView() {
                 && getViewBinder() != null
                 && boundActivityRef.getWindow() != null
                 && boundActivityRef.getWindow().getDecorView() != null) {
-            getViewBinder().clearBindingForViewAndChildren(getBoundActivity().getWindow().getDecorView().getRootView());
+            getViewBinder().clearBindingForViewAndChildren(getBoundActivity().getWindow().getDecorView());
```

Comment:
yes, if you're not using the BindableActivity, it has to be called manually from `onDestroy()` of the activity.